selenium_recaptcha_solver/api.py

```diff
@@ -1,3 +1,5 @@
+from typing import *
```

Comment:
Use a more specific import to avoid polluting the namespace; the ideal would be: from typing import Any.

Reply:
Sure